react/src/components/AgentDetailModal.tsx

```diff
@@ -0,0 +1,216 @@
+import { iSizeToSize } from '../helper';
+import { useResourceSlotsDetails } from '../hooks/backendai';
+import BAIModal, { BAIModalProps } from './BAIModal';
+import Flex from './Flex';
+import ResourceAvailableGageBar from './ResourceAvailableGageBar';
+import { AgentDetailModalFragment$key } from './__generated__/AgentDetailModalFragment.graphql';
+import { Button, Col, Row, Typography } from 'antd';
+import graphql from 'babel-plugin-relay/macro';
+import _ from 'lodash';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useFragment } from 'react-relay';
+
+type LiveStat = {
+  capacity: string;
+  current: string;
+  pct: string;
+  'stats.avg': string;
+  'stats.max': string;
+  unit_hint: string;
+};
+
+interface AgentDetailModalProps extends BAIModalProps {
+  agentDetailModalFrgmt?: AgentDetailModalFragment$key | null;
+  onRequestClose: () => void;
+}
+
+const AgentDetailModal: React.FC<AgentDetailModalProps> = ({
+  agentDetailModalFrgmt = null,
+  onRequestClose,
+  ...modalProps
+}) => {
+  const { t } = useTranslation();
+  const [resourceSlotsDetails] = useResourceSlotsDetails();
+  const agent = useFragment(
+    graphql`
+      fragment AgentDetailModalFragment on Agent {
+        live_stat
+        available_slots
+        occupied_slots
+        cpu_cur_pct
+        mem_cur_bytes
+      }
+    `,
+    agentDetailModalFrgmt,
+  );
+  const parsedLiveStat = JSON.parse(agent?.live_stat || '{}');
+  const parsedAvailableSlots = JSON.parse(agent?.available_slots || '{}');
+
+  return (
+    <BAIModal
+      {...modalProps}
+      centered
+      title={t('agent.DetailedInformation')}
+      onCancel={onRequestClose}
+      destroyOnClose
+      footer={[
+        <Button
+          onClick={() => {
+            onRequestClose();
+          }}
+          type="primary"
+        >
+          {t('button.Close')}
+        </Button>,
+      ]}
+    >
```

Comment:
FYI, this is another way to display only one button.
```suggestion
      cancelButtonProps={{style:{display:'none'}}}
      onOk={onRequestClose}
      okText={t('button.Close')}
```